www/viewgame

```diff
@@ -1970,27 +1983,36 @@ if ($mediantime > 60) {
 }
 
 
-function displayMedianTime($rounded_median_time, $alltimes) {
+function displayMedianTime($rounded_median_time, $times_with_notes, $adminPriv) {
     if ($rounded_median_time < 1) return;
     // Turn the total minutes into hours and minutes, and then text
     $official_estimated_time_text = convertTimeToText($rounded_median_time);
     echo "<details><summary>Estimated play time: ";
     echo $official_estimated_time_text;
-    $s = (count($alltimes) === 1) ? "" : "s";
-    echo " (based on " . count($alltimes) . " vote$s)</summary><br>";
+    $s = (count($times_with_notes) === 1) ? "" : "s";
+    echo " (based on " . count($times_with_notes) . " vote$s)</summary><br>";
     echo "Members voted for the following times for this game:";
     echo "<ul>";
-    foreach ($alltimes as $t) {
-        $time_vote_text = convertTimeToText($t);
-        echo "<li>$time_vote_text</li>";
+    foreach ($times_with_notes as $t) {
+        $time_in_text = convertTimeToText($t[time_in_minutes]);
+        $note = $t[time_note];
+        $voter = $t[userid];
+        $list_item = "<strong>$time_in_text</strong>";
+        if ($note != "") {
+            $list_item .= "—\"$note\"";      // If the voter added a note, display it
```

Comment:
Need to protect from HTML injection. @dfabulich might know how.

Reply:
Changed this